app/controllers/api/v1/rents_controller.rb

```diff
@@ -13,6 +13,7 @@ def index
       def create
         rent = Rent.new(creation_params)
         return invalid_params unless rent.save
+        send_mail_on_creation(rent)
```

Comment:
Remember to add specs to test that you're enqueuing the mails.
You need to add some code to the specs to setup sidekiq. You can google that or just let me know :). 

SPOILER: you should have something like in the controller test
`expect(Sidekiq::Worker.jobs.size).to eq(1)`